python/surfio/_surfio.pyi

```diff
@@ -0,0 +1,54 @@
+import os
+from typing import ClassVar
+
+import numpy
+import numpy.typing as npt
+
+class IrapHeader:
+    id: ClassVar[int] = ...  # read-only
+    ncol: int
+    nrow: int
+    rot: float
+    xinc: float
+    xmax: float
+    xori: float
+    xrot: float
+    yinc: float
+    ymax: float
+    yori: float
+    yrot: float
+    def __init__(
+        self,
+        ncol: int,
+        nrow: int,
+        xori: float = ...,
```

Comment:
Is those `...` actually required ?

Reply:
`= ...` shows that the parameter has a default value. The default value isn't really relevant for the type checker so it is omitted.